source/operations/checklists/hardware.rst

```diff
@@ -8,4 +8,131 @@ Hardware Checklist
    :local:
    :depth: 2
 
-Stub - detail Hardware Checklist Items
+Use the following checklist when planning the hardware configuration for a production, distributed MinIO deployment.
+
+Considerations
+--------------
+
+When selecting hardware for your MinIO implementation, take into account the following factors:
+
+- Expected amount of data in tebibytes to store at launch
+- Expected growth in size of data for at least the next two years
+- Number of objects by average object size
+- Average retention time of data in years
+- Number of sites to be deployed
+
+Hardware Requirements and Recommendations
+-----------------------------------------
+
+.. note:: 
+
+   For specific hardware suggestions, refer to https://min.io/product/reference-hardware.
+
+.. list-table::
+   :widths: auto
+   :width: 100%
+
+   * - :octicon:`circle`
+     - | Sufficient CPU cores to acheive performance goals
+       | MinIO recommends Dual Intel® Xeon® Scalable Gold CPUs (minimum 8 cores per socket)
+
+   * - :octicon:`circle`
+     - | Sufficient RAM to acheive performance goals
+       | MinIO recommends at least 128 GB of RAM 
+
+   * - :octicon:`circle`
+     - Four nodes or servers
+
+   * - :octicon:`circle`
+     - | SATA/SAS drives for capacity and NVMe SSDs for high-performance
+       | MinIO recommends a minimum of 8 drives per server
+
+   * - :octicon:`circle`
+     - | 25GbE network for capacity 
+       | 100GbE Network interface cards for high performance
+
+.. important:: 
+
+   Object storage operations are primarily throughput bound. 
+   MinIO takes full advantage of the modern hardware improvements such as AVX-512 SIMD acceleration, 100GbE networking, and NVMe SSDs when available.
+
+Hardware Tests to Complete
+--------------------------
+
+MinIO Diagnostics
+~~~~~~~~~~~~~~~~~
+
+Run the following command.
+If you have access to :ref:`SUBNET <minio-docs-subnet>`, you can upload the results there.
+
+.. code-block:: shell
+   :class: copyable
+
+   mc support diag ALIAS
+
+Drive Tests
+~~~~~~~~~~~
+
+Run these tests outside of MinIO to verify capabilities of your hardware.
+Test each drive independently on all servers to ensure they are identical in performance.
+Record the results for later reference.
+
+#. Uncached write test
+   
+    .. code-block::
+       :class: copyable
+
+
+       dd if=/dev/zero of=/mnt/driveN/testfile bs=128k count=80000 oflag=direct conv=fdatasync > dd-write-drive1.txt
+
+#. Uncached read test
+
+    .. code-block::
+       :class: copyable
+
+       dd if=/mnt/diskN/testfile of=/dev/null bs=128k iflag=direct > dd-read-drive1.txt
+
+   Replace ``diskN`` with the path for the disk you are testing.
+
+   If you need to create a sufficiently sized file, you can use the ``head`` command.
```

Comment:
We should state _why_ this is necessary.

IIRC there are both "small files" (<1MB) and "large files" (everything else) and "REALLY large files" in the 1GB+ range.

Not sure whether we can provide hard guidance here.

Reply:
Changed this line to the following:
```
Use a sufficiently sized file that mimics the primary use case for your deployment to get accurate read test results.
   
   Small files have a size < 1MB.
   Large files are from 1MB — 1GB.
   Really large files are > 1GB.

   You can use the ``head`` command to create a file to use.
   The following command example creates a 10 Gigabyte file called ``testfile``.
```